content/docs/core/configuration.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Configuration
+description: This page discusses configuration settings for Pomerium Core.
+keywords: [core configuration]
+sidebar_label: Configuration
+---
+
+# Configuration
+
+You can configure Pomerium using either a configuration file or [environment variables](https://en.wikipedia.org/wiki/Environment_variable). If using a configuration file, the following languages are supported:
+
+- [YAML](https://yaml.org/)
+- [JSON](https://www.json.org/json-en.html)
+- [TOML](https://toml.io/en/)
+
+(Our documentation always assumes a YAML configuration file.)
+
+You can use both environment variables and a configuration file. If a particular option is set using both an environment variable and a config file key, the environment variable will take precedence.
+
+:::tip
+
+Pomerium can hot-reload route configuration details, authorization policy, certificates, and other proxy settings.
+
+:::
+
+### Configuration syntax
+
+Both configuration file keys and environment variables are case sensitive.
+
+Configuration file keys are always lowercase. Environmental variables are identical to configuration file keys, except they are always uppercase.
+
+See the [Reference](/docs/reference) page for a comprehensive list of Pomerium's configuration settings.
+
+## All-In-One and Split-Service modes
+
+You can configure Pomerium using either All-In-One mode or Split Service mode.
+
+### All-In-One mode
+
+All-In-One mode means all of Pomerium's configuration settings are set in a single configuration file. You can use All-In-One mode when running Pomerium:
```

Comment:
I would prefer we pick one way to write "all-in-one" mode and use it consistently. By my count the existing docs have:
- 49 instances of "all-in-one"
- 3 instances of "All-in-One"
- 1 instance each of "All-in-one" and "All-in-One"

Based on the existing usage "All-In-One" looks out of place to me here.

Reply:
OK. I left a comment earlier which explains my position on this. But, I've updated this page so that is uses "All-in-One".

Reply:
For the record, I'm saying "all-in-one" (lowercase) would be most consistent with the existing documentation. If you don't think we should aim to be consistent I won't argue the point any further.